checks/evaluation/code_review.go

```diff
@@ -54,15 +57,37 @@ func CodeReview(name string, dl checker.DetailLogger, r *checker.CodeReviewData)
 		switch {
 		case isReviewed && isBotCommit:
 			foundBotReviewActivity = true
+			botReviewActivityRevisionIDs = append(botReviewActivityRevisionIDs, cs.RevisionID)
 		case isReviewed && !isBotCommit:
 			foundHumanReviewActivity = true
 		case !isReviewed && isBotCommit:
 			nUnreviewedBotChanges += 1
+			unreviewedBotRevisionIDs = append(unreviewedBotRevisionIDs, cs.RevisionID)
 		case !isReviewed && !isBotCommit:
 			nUnreviewedHumanChanges += 1
+			unreviewedHumanRevisionIDs = append(unreviewedHumanRevisionIDs, cs.RevisionID)
 		}
 	}
 
+	// Let's include non-compliant revision IDs in details
+	if len(botReviewActivityRevisionIDs) > 0 {
```

Comment:
same comment as above. Warn or Debug?

In the past, we've used Warn for things that can be remediated, and debug for things that provide additional information.

Reply:
Do Debug messages show up in CLI? In my limited experiment, it didn't show up so I decided to use Warn to explain why the user is getting a low score in this category.

Reply:
you would need to run with both `--show-details` and `--verbosity debug`

Reply:
Spoke with Raghav and the code review changes are fine here, we will work around them in the other PR.

I still have a question around collecting the reviewed bot PR

Reply:
Thanks @spencerschrock  for flatting the reviewed bot issue, I have removed that part.